engine/src/lib.rs

```diff
@@ -842,6 +842,16 @@ mod contract {
             .set_eth_connector_contract_account(&args.account);
     }
 
+    #[no_mangle]
+    pub extern "C" fn disable_legacy_nep141() {
+        let io = Runtime;
+        io.assert_private_call().sdk_unwrap();
```

Comment:
Probably better to make it owner-only instead of private. Not that it matters too much since I think for `aurora` the owner is the engine itself.